src/components/ComponentPages/UploadFiles/index.tsx

```diff
@@ -114,11 +125,16 @@ const UploadFiles = () => {
     dragBoxHide();
     showFiles();
     uploadOptionsHide();
+    /////
```

Comment:
If possible, please remove this uncommented code

Reply:
done